src/databricks/labs/ucx/azure/credentials.py

```diff
@@ -288,6 +298,14 @@ def _create_storage_credentials_for_storage_accounts(self) -> list[StorageCreden
                 validation_results = self._storage_credential_manager.validate(storage_credential_info, url)
             except BadRequest:
                 logger.warning(f"Could not validate storage credential {storage_credential_info.name} for url {url}")
+            except (PermissionDenied, NotFound):
+                logger.error(
+                    "Error validating all storage credentials. Please check the error message and tr again."
```

Comment:
Similar update as above

Reply:
access_connectors returns list[tuple[AccessConnector, str]] and i use the AccessConnector data class to get the resource id of the access connector which is passed to delete the access connector